app/src/main/java/ch/blinkenlights/android/vanilla/TrackDetailsDialog.java

```diff
@@ -0,0 +1,106 @@
+package ch.blinkenlights.android.vanilla;
```

Comment:
Could you add the usual GPL-3 copyright boilerplate? (Obviously with your name instead of mine).

LGTM apart from that - thanks a lot!